tap_chargebee/client.py

```diff
@@ -29,6 +29,9 @@ def __init__(self, config, api_result_limit=100, include_deleted=True):
         self.include_deleted = include_deleted
         self.user_agent = self.config.get('user_agent')
 
+        if self.config.get('include_deleted') in ['false','False', False]:
+            self.include_deleted = False
```

Comment:
This seems odd. Why is this saved onto the client and never referenced?

Reply:
We haven't added this parameter, it was there with the default value `true`. We just updated it for the particular scenario.  
And it got referred at line no 49 inside `get_params` function:  `params['include_deleted'] = self.include_deleted`